rise-data.html

```diff
@@ -105,6 +111,83 @@
 
         // log usage
         this.$.logger.log(BQ_TABLE_NAME, params);
+      },
+
+      /**
+       * Cache the data
+       *
+       * @param {String} key The key to identify the item of data
+       * @param {Object} data The data to cache
+       * @param {Boolean} ts Optionally include a timestamp to cache
+       */
+      saveItem: function(key, data, ts) {
+        var cacheObj = {};
+
+        if (this._pingReceived && key && data) {
+          if (!this._isCacheRunning) {
+            if (supportsLocalStorage()) {
+              cacheObj.data = data;
+
+              if (ts) {
+                cacheObj.timestamp = moment().format();
```

Comment:
@stulees It's unfortunate, but I guess we'd better leave it for now to avoid any incompatibilities.